code/controllers/subsystem/tickets/tickets.dm

```diff
@@ -328,6 +347,8 @@ SUBSYSTEM_DEF(tickets)
 
 //Return the ticket state as a colour coded text string.
 /datum/ticket/proc/state2text()
+	if(ticket_converted)
```

Comment:
wouldn't this also make it show up as converted when it's resolved etc?

Reply:
It `return`s the status text, so it won't get to the others if it's converted.

Reply:
Aye so resolved tickets still are labeled converted. Should be fine due to the filters but does appear a bit weird